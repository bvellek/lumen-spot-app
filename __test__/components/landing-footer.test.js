import React from 'react';
import renderer from 'react-test-renderer';

import LandingFooter from '../../public/js/components/landing-footer';

describe('<LandingFooter />', () => {
  it('it renders correctly', () => {
    const component = renderer.create(
      <LandingFooter />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
