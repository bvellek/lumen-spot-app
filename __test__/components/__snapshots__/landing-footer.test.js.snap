// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingFooter /> it renders correctly 1`] = `
<footer>
  <p>
    Designed and Developed by 
    <a
      className="sliding-middle-out"
      href="https://github.com/bvellek"
      target="_github"
    >
      BenÂ Vellek
    </a>
  </p>
</footer>
`;
